alacritty_terminal/src/term/mod.rs

```diff
@@ -949,9 +987,9 @@ impl<T> Term<T> {
 
         if is_block {
             for line in (end.line + 1..=start.line).rev() {
-                res += &(self.line_to_string(line, start.col..end.col) + "\n");
+                res += &(self.line_to_string(line, start.col..end.col, start.col.0 == 0) + "\n");
```

Comment:
Could you explain to me the idea behind including wrap here? I feel like we shouldn't do it?

Reply:
If we have a block selection that includes part of a wide char at the right, but which doesn't start at the left side of the terminal (in column 0), we need to expand across the wide char.

Reply:
But it should be `start.col.0 != 0`.

Reply:
not it make more sense.